cvrf2csaf/cvrf2csaf.py

```diff
@@ -7,8 +7,21 @@
 
 from .common.utils import get_config_from_file, store_json, critical_exit
 
-from .section_handlers.document_tracking import DocumentTracking
+from .section_handlers.document_acknowlegments import Acknowledgments
+from .section_handlers.document_aggregate_severity import AggregateSeverity
+from .section_handlers.document_category import DocumentCategory
```

Comment:
Where does this come from? I cannot see any DocumentCategory in the specification:

http://docs.oasis-open.org/csaf/csaf-cvrf/v1.2/cs01/schemas/cvrf.xsd

Reply:
Ah I see, naming after CSAF.. see the point 2) in the comment below: